skimage/filters/rank/generic.py

```diff
@@ -127,6 +129,30 @@ def _apply_vector_per_pixel(func, image, selem, out, mask, shift_x, shift_y,
     return out
 
 
+def default_selem(func):
```

Comment:
Not a problem, but please make it a private function by inserting a leading underscore.